collector/collector.go

```diff
@@ -691,6 +668,17 @@ func (c *collector) CollectMetrics(mts []plugin.Metric) ([]plugin.Metric, error)
 		return nil, fmt.Errorf("No metrics found")
 	}
 
+	// add labels as tags to metrics
+
+	for i := range metrics {
+		rid := metrics[i].Namespace[2].Value
+		if rid != "root" {
+			// add labels only for docker cointainer, skip the host
+			metrics[i].Tags = c.containers[rid].Specification.Labels
```

Comment:
To satisfied both of case, I put such code:
```
if len(metrics[i].Tags) == 0 {
 				metrics[i].Tags = c.containers[rid].Specification.Labels
 			} else {
 				// adding labels one by one to existing tags
 				for lkey, lval := range c.containers[rid].Specification.Labels {
 					metrics[i].Tags[lkey] = lval
 				}
 			}
```
what means that this `tags` is overrided only if it is empty - currently this case happens every time, but such if...else statement might be helpful in the future not to lost some tags added before.